testsuite/helloworld/tests/bf_rte_low/test.yml

```diff
@@ -0,0 +1,5 @@
+language: BF
+time: 1
+memory: 4
```

Comment:
`mmap` rounding to page size is not documented and is broken in some cases, so let's just specify a reasonable limit here.

Reply:
Upgraded 4 to 64.